### PR TITLE
Integrated repo aem-stack-manager-cloud to aem-aws-stack-builder #63

### PR DESCRIPTION
**Pull request by mbloch1986**
Integrated the repo aem-stack-manager-cloud to aem-aws-stack-builder with necessary scripts.

Using:

make [create-stack-manager|delete-stack-manager]
make [create-ssm-documents|delete-ssm-documents]
make [create-snapshots-purge|delete-snapshots-purge]

### CODE REVIEW

File: cloudformation/apps/snapshots-purge.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (cliffano) [inline cloudformation/apps/snapshots-purge.yaml]: Move this CloudFormation to `cloudformation/apps/utilities.yaml` .

File: cloudformation/apps/instance-profiles.yaml [hunk 2]
```diff
@@ -10,6 +10,18 @@ Parameters:
       Type: String
```
Comment (cliffano) [inline cloudformation/apps/instance-profiles.yaml]: Move the stack-manager specific instance profiles to `cloudformation/apps/stack-manager/instance-profiles.yaml` . Ditto for playbook.

File: cloudformation/apps/stack-manager.yaml [hunk 3]
```diff
@@ -0,0 +1,381 @@
+---
```
Comment (cliffano) [inline cloudformation/apps/stack-manager.yaml]: Move this to `cloudformation/apps/stack-manager/main.yaml` .

File: examples/user-config/full-set/sandpit-apps.yaml [hunk 4]
```diff
@@ -65,6 +65,10 @@ library:
   aem_orchestrator_version: 1.0.1-SNAPSHOT
   aem_password_reset_version: 1.0.1
   oak_run_version: 1.4.15
+  aem_stack_manager_lambda_version: 0.0.1
```
Comment (cliffano) [inline examples/user-config/full-set/sandpit-apps.yaml]: Same as the default config.

File: scripts/stack-manager/generate.sh [hunk 5]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (cliffano) [inline scripts/stack-manager/generate.sh]: Move all stack-manager scripts into stack manager cloud repo, published as a version artifact (along with the lambda functions).
The stack manager creation playbook should then download the artifact, unpack, and prepare the staged files accordingly.

File: ansible/inventory/group_vars/apps.yaml [hunk 6]
```diff
@@ -87,6 +87,10 @@ library:
   aem_orchestrator_version: 1.0.0
   aem_password_reset_version: 1.0.1
   oak_run_version: 1.4.15
+  aem_stack_manager_lambda_version: 0.0.1
```
[the remark this answers is not in the document]
Comment (mbloch1986) [reply ansible/inventory/group_vars/apps.yaml]: Changed to: 
- aem_stack_manager_version: 2.0.0

File: examples/user-config/full-set/sandpit-apps.yaml (hunk 4 above)
Comment (mbloch1986) [reply examples/user-config/full-set/sandpit-apps.yaml]: Changed to: 
- aem_stack_manager_version: 2.0.0

**Pull request merged**